docs/getting-started-guides/windows/index.md

```diff
@@ -1,149 +1,256 @@
 
 Each Window Server node should have the following configuration:
 
 The following diagram illustrates the Windows Server networking setup for Kubernetes using Upstream L3 Routing Setup:
 ![K8s Cluster using L3 Routing with ToR](UpstreamRouting.png)
 
 #### Host-Gateway Topology
-This topology is similar to the Upstream L3 Routing topology with the only difference being that static IP routes are configured directly on each cluster node and not in the upstream ToR. Each node uses a local 'l2bridge' network with a Pod CIDR assigned as before and has routing table entries for all other Pod CIDR subnets assigned to the remote cluster nodes.
+This topology is similar to the Upstream L3 Routing topology with the only difference being that static IP routes are configured directly on each cluster node and not in the upstream ToR. Each node uses a local 'l2bridge' network with a pod CIDR assigned as before and has routing table entries for all other pod CIDR subnets assigned to the remote cluster nodes.
+
+#### Using OVN with OVS
+The following diagram gives a general overview of the architecture and interaction between components:
+
+![Overlay using OVN controller and OVS Switch Extension](ovn_kubernetes.png)
 
-#### Overlay using OVN controller and OVS Switch Extension
+(The above image is from https://github.com/openvswitch/ovn-kubernetes#overlay-mode-architecture-diagram)
 
-TODO
+Due to its architecture, OVN has a central component which stores your networking intent in a database. Other components i.e. kube-apiserver, kube-controller-manager, kube-scheduler etc. can be deployed on that central node as well. 
 
 ## Setting up Windows Server Containers on Kubernetes
-To run Windows Server Containers on Kubernetes, you'll need to set up both your host machines and the Kubernetes node components for Windows and depending on your network topology, setup Routes for Pod communication on different nodes.
+To run Windows Server Containers on Kubernetes, you'll need to set up both your host machines and the Kubernetes node components for Windows. Depending on your network topology, routes may need to be set up for pod communication on different nodes.
 
 ### Host Setup
 
-**Linux Host Setup**
+#### For 1. Upstream L3 Routing Topology and 2. Host-Gateway Topology
+
+##### Linux Host Setup
 
 1. Linux hosts should be setup according to their respective distro documentation and the requirements of the Kubernetes version you will be using. 
-2. Configure Linux Master node using steps [here](https://github.com/Microsoft/SDN/blob/master/Kubernetes/HOWTO-on-prem.md)
+2. Configure Linux Master node using steps [here](https://github.com/Microsoft/SDN/blob/master/Kubernetes/HOWTO-on-prem.md#prepare-the-master)
```

Comment:
Will update link to be here: https://github.com/gkudra-msft/Virtualization-Documentation/blob/kubernetes-docs/virtualization/windowscontainers/kubernetes/creating-a-linux-master.md

Reply:
Similar to above, I would like to see the changes here rather than on a fork: https://github.com/MicrosoftDocs/Virtualization-Documentation

Reply:
i agree with @alinbalutoiu

Reply:
In my opinion since this is the official documentation, it should point to its internals i.e.: `https://kubernetes.io/docs/getting-started-guides/scratch/ `.

Reply:
Yes, we will be merging the forked changes to original repo